### DISCUSSION TITLE
老杨：TMD崩坏还在追我

### DISCUSSION THREAD

**Discussion by xingyun-kb** [share121/inter-knot#4887] [崩坏3]
![mmexport1726371542016](https://github.com/user-attachments/assets/e7eac225-b27b-4c62-bca6-4e39d1a9d77c)

**Comment by K-k423** [comment]
后面还有一版
![1726318508115](https://github.com/user-attachments/assets/afc44ce9-382a-40b1-b7a0-014dcc57f4d3)
<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/b171cc5b6c8d8b6257a3bf8163fb2680_567796524796359289.png" alt="阿姬得意" width="75" >
